models/labels/labels_all_schema.yml

```diff
@@ -3,8 +3,11 @@ version: 2
 models:
   - name: labels_all
     tests:
-    - unique:
-        column_name: "(address || '-' || name || '-' || blockchain::string)"
+      - dbt_utils.unique_combination_of_columns:
```

Comment:
this one is more for @soispoke fyi so we can continue to enhance schema files to use the cleaner format